extraDevfileEntries.yaml

```diff
@@ -0,0 +1,18 @@
+schemaVersion: 1.0.0
+samples:
```

Comment:
@elsony should samples be the top level or a type(type can be stack or sample)?

Proposal 2 has it in the top level and the index output has it as type https://docs.google.com/document/d/1tbzZK9aWtpw5AtoZmUl1Jy8x8w9N5hhjXu3c26VYiYY/edit#heading=h.6w072oor8mro

Reply:
@maysunfaisal When I wrote the proposal, I started with having both samples and stacks under the same list in the `extraDevfileEntries.yaml`. However, I find that I need to add an extra non-functional parent, e.g. `devfiles:` at the top level. I find in that file, it is cleaner to just separate out those two lists. 

Note that I keep `type` field in index.json since the registry just contains a list of devfile entires. Having them in the same list will be naturally having one single list to contain all devfiles in the query in the returned json object.

Do your think it works better to have the type instead of two separate lists in that file? I don't feel strongly about both approaches so I am open to suggestions.

Reply:
two lists is fine